packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/ipv4/ipv4.constant.js

```diff
@@ -1,5 +1,6 @@
 import {
   TRACKING_PREFIX,
+  DASHBOARD,
```

Comment:
Since you don't use the DASHBOARD constant to build another constant in this file, you can directly import the one from the parent ip-ip-agoraOrder.constant.js in the controller

```suggestion
```